index.html

```diff
@@ -0,0 +1,64 @@
+<!-- landing page for personal portfolio -->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title> Tamiko Terada </title>
+    <link rel="stylesheet" type="text/css" href="stylesheets/main.css">
+    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css">
+  </head>
+
+  <body>
+    <a href="html/menu.html" target="_parent"><button autofocus id="test"> </button></a>
+    <!-- header | name, image, banner, button-link -->
+    <header>
+      <div class="left-bar">
+        <p class="top-bar">Tamiko Terada</p>
+      </div>
+      <nav><a href="menu.html">menu</a></nav>
+    </header>
+
+    <div id="header-image"></div>
+    <section>
+      <div class="box-green">
+        <p class="centered"><a class="button" href="https://www.linkedin.com/in/tamikoterada/" target="_blank">TAMIKO has skills.</a>
+        </p>
+      </div>
+    </section>
+
+    <div class="box-green"></div>
+
+    <main>
+      <p class="plain-bar">Press ENTER to size up the situation</p>
+      <!-- left column -->
+      <section class="box">
+        <div id="column-one">
+          <ul>
+            <li> Start date: </li>
+            <li> Weather: </li>
+            <li> Food: </li>
+            <li> Next landmark: </li>
+          </ul>
+        </div>
+        <!-- right column -->
+        <div id="column-two">
+          <ul>
+            <li> February 1, 2018 </li>
+            <li> Seattle rain </li>
+            <li> lots of snacks </li>
+            <li> employment </li>
+          </ul>
+        </div>
+      </section>
+      <!-- banner -->
+    <p class="plain-bar">Press SPACE BAR TO CONTINUE</p>
+    </main>
+    <!-- footer -->
+    <footer>
+      <p>Disclaimer: This portfolio site was made for educational purposes only in homage to my first encounter with a computer. Oregon Trail was created by Don Rawitsch, Bill Heinemann, and Paul Dillenberger with the Minnesota Educational Computing Consortium (MECC) in 1974. The version of the game referenced here is no longer sold and falls under the DMCA exemptions of 2006.
```

Comment:
Generally good practice not to let lines get more than 80 characters (give or take), and formatting either all on one line (if it fits) or all staggered makes html more readable: 
```
<p> Some paragraph. </p>
``` 
OR
```
<p>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et 
  dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex 
  ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat 
  nulla pariatur.
</p>
```
(your have 1/2 of each here)